src/FSharp.Analyzers.Cli/Program.fs

```diff
@@ -11,14 +11,18 @@ open Microsoft.CodeAnalysis.Sarif.Writers
 open Ionide.ProjInfo
 
 type Arguments =
-    | Project of string list
-    | Analyzers_Path of string list
-    | Fail_On_Warnings of string list
-    | Ignore_Files of string list
-    | Exclude_Analyzer of string list
-    | Report of string
-    | FSC_Args of string
-    | Verbose
+    | [<Unique>] Project of string list
```

Comment:
does `Unique` require only a single instance of each of these flags? It's often nicer/easier for users to specify the same flag multiple times, aggregating all of the values of each time the flag is provided. 

```console
> app --project myproject.fsproj --project anotherproject.fsproj
```

is the same as

```console
> app --project myproject.fsproj,anotherproject.fsproj
```